### PR TITLE
blog: gator tank competition

### PR DESCRIPTION
**Pull request by danielwildsmith**

### CODE REVIEW

**Comment by danielwildsmith** [comment]
@zeim839 thoughts?

**Comment by danielwildsmith** [comment]
i should probably include the date of the event, anything else u can think of?

**Comment by zeim839** [comment]
Will take a look at this tomorrow

**Comment by zeim839** [comment]
SEOmaxxing

File: src/content/2024-10-08-gator-tank-competition.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+slug: "/2024-gator-tank"
+date: "2024-10-09"
+title: "OSC Competes in and Wins UF Gator Tank 2024"
```
Comment (zeim839) [inline src/content/2024-10-08-gator-tank-competition.md:4]: On second though, can you please change this to "OSC Competes in UF Gator Tank 2024"

**Comment by danielwildsmith** [comment]
i gotchu. this is incredible lmao. might need to change your name to Homer if you're capable of writing epics like this

**Pull request merged**